### ISSUE TITLE
Map Indexer Crashes Every Time

### ISSUE THREAD

**Issue by Vegas** [ALiVEOS/ALiVE.OS#283]
So, I have indexed several maps for ALiVE for both my unit, and to provide to the ALiVE forums for others to download. However, my tools seem to be suddenly broken and I am unable to index (or reindex) even vanilla maps like Stratis.

**Issue**
- Indexer tools crash every single time as soon as I open the ALiVE menu (app data), and hit the map indexer

**Repro Mission and RPT**
https://mega.nz/#!8pdVBSLZ!okAcPQAqeOvCuhqriCKBBqkhz9E7nMqeHTe4JJKWAbc

- I know we have had some issues with Windows 7 OS in the past, I am hoping this is not the issue because I dislike Windows 10 and it would suck to upgrade just to index maps.
- I have also tried changing the OS from 7 to 10 on the Indexer Module itself, still crashes.
- I am also running the main 32-bit branch of Arma 3. I did not test this with 64-bit on dev branch.

**My PC Specs**
OS: Windows 7 Home Premium 64-bit (6.1, Build 7601)
CPU: Intel i7-4790k
RAM: 32GB DDR3
GPU: NVIDIA GeForce GTX 1070
And I'm running all exe's and dll's unblocked.

Any help on this would be appreciated. I want to get back to indexing maps..

**Comment by marceldev89** [comment]
Did anything change with your setup since you last successfully indexed a map?

**Comment by Vegas** [comment]
I have not changed anything as far as my setup goes. There has been one Arma 3 update, and a bunch of updates to ALiVE (obviously).

**Here are the logs from indexing folder:**
https://mega.nz/#!A0U1RA7L!j6wWhY_-2dBu6JIrQUz-wLNwIGnzksqXPH_sX8enUko

Appreciate the help as always.

**Comment by marceldev89** [comment]
It seems like you're crashing at the same point as where a few others crash. Try running
```
DeWrp.exe -O "C:\Program Files (x86)\Steam\steamapps\common\Arma 3\Addons\map_stratis.pbo" > "C:\Program Files (x86)\Steam\steamapps\common\Arma 3\@ALiVE\indexing\stratis\x\alive\addons\fnc_strategic\indexes\objects.stratis.sqf"
```
with `cmd.exe` and check the contents of `C:\Program Files (x86)\Steam\steamapps\common\Arma 3\@ALiVE\indexing\stratis\x\alive\addons\fnc_strategic\indexes\objects.stratis.sqf` after that.

**Comment by Vegas** [comment]
After trying to run the above command through cmd.exe, I got this error.

DeWrp.exe - Entry Point Not Found
---------------------------
The procedure entry point ?wrp_GetStruct@DeWrp@@QEAAPEAUWarpStruct@@XZ could not be located in the dynamic link library DePbo64.dll. 

Not really sure what this is or how to resolve it? I tried removing all quotation marks from your posted above and all I get when trying to run that is "Access is denied".
